Translate/En/21-08.md

```diff
@@ -1,118 +1,121 @@
+As I looked up, a red knight, whose both eyes filled with horrible hatred caused the helmet to seemingly shake, started gripping the long sword with both hands.
 
-A sword is thrust down, with a stinging screech.
+A sword is thrust down with a stinging screech.
 
-There is no power to fight anymore, but Asuna stares at the steel iron trying to penetrate her, at least not whether the eyes are closed.
+There is no power within her to fight anymore, but Asuna still stares at the steel blade as it attempts to pierce her, at least her eyes are not closed(??).
 
-Kin.
+Kin--!
 
-Hard metal sound. Orange spark.
+Hard metal clashing sound is heard. Orange spark's are emitted.
 
-The knight's sword bounced off in the air as though it had been flicked by a colorless and transparent blade.
+The knight's sword bounced off into the air, as though it had been knocked away by a colorless and transparent blade.
 
-"Wow ...?"
+"Wow...?"
 
-A knight who revealed a disconcerting voice will swing down the sword again. But the sparks scatter again, the sword does not reach Asuna.
+The knight who revealed a disconcerting voice swings down the sword again. However, sparks scatters again, the sword is unable to reach Asuna.
 
-For the third time, the fourth time, the result is the same.
+Even for the third time, the fourth time, the results are still the same.
 
-The fifth attack was not done. Solterina, who came running to Asuna, played the Red Knight greatly with the sword skill "Torrent" for knockback using the handle of the large sword. While helping Asuna, Solterina asked surprise in a hidden voice.
+Before the fifth attack was executed, Solterina, who came running to Asuna, pushed(?) the Red Knight back(?) greatly with the sword skill "Torrent", producing knockback using the handle of the large sword. While helping Asuna up, Solterina asked surprisingly in a small voice.
 
-"The "sword of mind" now ... ... Asuna !?"
+"Asuna...was that the "Sword of Mind" just now...!?"
 
-"Incarnation ... ...?"
+"What's the incarnation...?"
 
-Shake head a little to the word you listen for the first time.
+I shoke my head a little towards the word I heard for the first time.
 
 "No, it is not me."
 
-"Well ...... Renri is ... ...."
+"Well, Renri is..."
 
-As Asuna chased the line of sight of Solterina turning around while saying that, the boy knight who was seriously injured was flying instructions to the troops to intercept the army of the approaching red knight, and he said that he was able to afford to keep Asuna very much I do not think so.
+As Asuna followed Solterina's line of sight as she turns around while saying that, the male knight, who was terribly injured was giving instructions to the troops to intercept the army of the approaching red knight, said he don't think he is able to afford to keep an eye on Asuna any much longer. (??)
```

Comment:
...and by the look of it Asuna ensures sure that Renri has no time/power left for shooting protective incarnations around.